pkg/controllers/controllers.go

```diff
@@ -70,7 +70,7 @@ func NewControllers(
 		metricspod.NewController(kubeClient),
 		metricsprovisioner.NewController(kubeClient),
 		counter.NewController(kubeClient, cluster),
-		consistency.NewController(clock, kubeClient, recorder, cloudProvider),
+		consistency.NewMachineController(clock, kubeClient, recorder, cloudProvider),
```

Comment:
Why are we not using `NewNodeClaimController`?

Reply:
This would enforce that we have `NodeClaims` that are deployed and are shipped as part of the chart. That'll happen when we are ready to release the changes all together. For now, we are just staging out the changes and having it work with the older API version.